docs/sources/configure-client/grafana-agent/ebpf/configuration/_index.md

```diff
@@ -0,0 +1,122 @@
+| `collect_interval`        | `duration`               | How frequently to collect profiles                           | `15s`   | no       |
+| `sample_rate`             | `int`                    | How many times per second to collect profile samples         | 97      | no       |
+| `pid_cache_size`          | `int`                    | The size of the pid -> proc symbols table LRU cache          | 32      | no       |
+| `build_id_cache_size`     | `int`                    | The size of the elf file build id -> symbols table LRU cache | 64      | no       |
+| `same_file_cache_size`    | `int`                    | The size of the elf file -> symbols table LRU cache          | 8       | no       |
+| `container_id_cache_size` | `int`                    | The size of the pid -> container ID table LRU cache          | 1024    | no       |
+| `collect_user_profile`    | `bool`                   | A flag to enable/disable collection of userspace profiles    | true    | no       |
+| `collect_kernel_profile`  | `bool`                   | A flag to enable/disable collection of kernelspace profiles  | true    | no       |
+| `demangle`                | `string`                 | C++ demangle mode. Available options are: `none`, `simplified`, `templates`, `full` | `none` | no |
+
+## Supported languages
+
+The `pyroscope.ebpf` component supports the following languages:
+
+- Go
+- Rust
+- C/C++ with frame pointers enabled
+- Python
+
+## Sending data Grafana Cloud Profiles
+
+When sending to Grafana Cloud Profiles, you can use the following `pyroscope.write` component configuration which makes uses of environment variables:
+
+```river
+pyroscope.write "endpoint" {
+    endpoint {
+        basic_auth {
+            password = env("GC_PASSWORD")
+            username = env("GC_USER")
+        }
+        url = env("GC_URL")
+    }
+}
+```
+
+Ensure that you have appropriately configured the `GC_URL`, `GC_USER`, and `GC_PASSWORD` environment variables.
+
+## Profile collecting behavior
+
+The `pyroscope.ebpf` component collects stack traces associated with a process running on the current host.
+You can use the `sample_rate` argument to define the number of stack traces collected per second. The default is 97.
+
+The following labels are automatically injected into the collected profiles if you have not defined them. These labels
+can help you pin down a profiling target.
+
+| Label              | Description                                                                                                                      |
+|--------------------|----------------------------------------------------------------------------------------------------------------------------------|
+| `service_name`     | Pyroscope service name. It's automatically selected from discovery meta labels if possible. Otherwise defaults to `unspecified`. |
```

Comment:
```suggestion
| `service_name`     | Pyroscope service name. It's automatically selected from discovery meta labels, if possible. Otherwise, it defaults to `unspecified`. |
```